src/js/vertical-slide.js

```diff
@@ -1,49 +1,48 @@
 // Variables
+const hostName = location.hostname;
 const sliderContext = document.querySelector('#slider-context');
 const sliderPagination = document.querySelector('#slider-pagination');
 const shortCodeId = sliderContext.getAttribute('data-shortcode-id');
 const shortCodeSlug = sliderContext.getAttribute('data-shortcode-slug');
 const slides = [];
+const counterObj = {
+  value: 0
+};
```

Comment:
If `counterObj` is only ever going to be a single integer, should we just make the object a simple integer?

```suggestion
let counterObj = 0;
```

Reply:
Well, the problem arises when I try to pass the counter into the function as an argument and update it there. Since the counter is a central variable, it gets accessed by multiple functions. For example, when the user scrolls down or up, the counter will be changed. In the future, when I add touch behavior or key up, key down behavior, they will also update the same counter. Passing it as a primitive variable won't update the reference. Therefore, I needed to store it in an object to update the reference every time the counter changes.

Reply:
Is there a reason we're passing the counter as a parameter in some cases and just modifying it in others? Should we not be consistent in the way we modify the value throughout the code to avoid confusion in the future?